onmt/modules/PDBiEncoder.lua

```diff
@@ -110,6 +110,23 @@ function PDBiEncoder:maskPadding()
   self.layers[1]:maskPadding()
 end
 
+-- size of context vector
+function PDBiEncoder:contextSize(sourceSize, sourceLength)
```

Comment:
contextSize is used in decoder to have size of the context size outside of the encoder which varies for pdbiencoder